incubator/hnc/internal/reconcilers/hierarchy_config.go

```diff
@@ -43,6 +43,8 @@ import (
 	"sigs.k8s.io/multi-tenancy/incubator/hnc/internal/stats"
 )
 
+const propagatePrefix = "propagate.hnc.x-k8s.io/"
```

Comment:
IMO this should go into the API package, since the prefix is part of the API.

Reply:
Removed